.github/workflows/ci.yml

```diff
@@ -42,13 +44,6 @@ jobs:
           distribution: temurin
           java-version: 8
 
-      - name: Setup Java (temurin@17)
```

Comment:
Is this removal intentional ?

Reply:
Oh apologies, you were testing on 17 here as well. Good catch, easy to add back.